packages/components/src/components/InputBox/InputBox.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { InputBox } from "./InputBox";
+
+export default {
+  component: InputBox,
```

Comment:
Is it worth showing InputBox in Storybook? Would developers need or care about InputBox since Input is what they'd be using?

Reply:
Not super important, but we have a lot of snapshots so also not hurting anything. It will help us nail down any style bugs since those could potentially be in Input or InputBox.